bootstrap/namespaces/helmfile/values/namespaces-sc.yaml.gotmpl

```diff
@@ -1,15 +1,58 @@
 namespaces:
 - name: cert-manager
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: v1.25
+    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce-version: v1.25
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/warn-version: v1.25
 - name: dex
```

Comment:
What about the namespaces without labels, such as Dex? Can you clarify your choice?